src/include/sof/gdb/gdb.h

```diff
@@ -3,7 +3,7 @@
 
 #define GDB_BUFMAX 256
 #define GDB_NUMBER_OF_REGISTERS 64
-
+#define DISABLE_LOWER_INTERRUPTS_MASK ~0x1F
```

Comment:
This looks platform specific ? If so, it should be in platform.h

Reply:
Yes, I will move it there.